code/_onclick/hud/alert.dm

```diff
@@ -202,30 +209,36 @@ or shoot a gun to move around via Newton's 3rd Law of motion."
 /obj/screen/alert/nocell
 	name = "Missing Power Cell"
 	desc = "Unit has no power cell. No modules available until a power cell is reinstalled. Robotics may provide assistance."
+	icon_state = "nocell"
 
 /obj/screen/alert/emptycell
 	name = "Out of Power"
 	desc = "Unit's power cell has no charge remaining. No modules available until power cell is recharged. \
 Reharging stations are available in robotics, the dormitory's bathrooms. and the AI satelite."
+	icon_state = "emptycell"
 
 /obj/screen/alert/lowcell
 	name = "Low Charge"
 	desc = "Unit's power cell is running low. Reharging stations are available in robotics, the dormitory's bathrooms. and the AI satelite."
```

Comment:
Unrelatedly this is dongs because it doesn't support any other maps seriously don't do this